### ISSUE TITLE
the test results on HICO-Det and V-COCO after training 150 epochs are very low, it doesn't seem to converge.

### ISSUE THREAD

**Issue by hustpenghuan** [bbepoch/HoiTransformer#26]
I used 32 A100 gpu, batch size per gpu is 6, learing rate is 12e-4, learning rate of backbone is 12e-5, other settings are same.How can i get the correct result? Any helpful advise is appreciated!
![image](https://user-images.githubusercontent.com/45789117/126280037-8b6cae1a-089e-4655-bf92-faab236e3508.png)

**Comment by bbepoch** [comment]
I suggest you  do experiments with the same setting at first. And change the settings only when you get the desired results. Or there may be some environment related problems to solve.
